codecs.go

```diff
@@ -12,8 +12,8 @@ type CodecType int
 const (
 	// RSGF8 represents Reed-Solomon codecType with an 8-bit Finite Galois Field (2^8)
 	RSGF8       CodecType = iota
-	LeopardFF8  CodecType = 1
-	LeopardFF16 CodecType = 2
+	LeopardFF8  CodecType = iota
+	LeopardFF16 CodecType = iota
```

Comment:
Can this be simplified s.t. only the first line says iota?
Same below.

Reply:
These cannot, since `CodecType` is a custom type. The compiler needs an actual value assigned. The other ones are just `int`s, so they can.

Reply:
Fixed in 888fd64.